### PR TITLE
Connection count

### PR DESCRIPTION
**Pull request by amazingandyyy**
add connection count to bottomBar

### CODE REVIEW

File: src/components/BottomBar/index.js [hunk 1]
```diff
@@ -38,12 +33,11 @@ export default class BottomBar extends Component {
 
   render() {
     const { classes } = this.props;
-    const { syncBlockNum, syncBlockTime } = this.props.store.global;
-
+    const { syncBlockTime, syncBlockNum, peerNodeCount } = this.props.store.global || { syncBlockTime: 1, syncBlockNum: 1, peerNodeCount: 1 };
```
Comment (dwalintukan) [inline src/components/BottomBar/index.js]: lets not set the defaults here if they are undefined. the store defaults should be sufficient enough.

File: src/components/BottomBar/index.js [hunk 2]
```diff
@@ -16,10 +15,6 @@ import { getShortLocalDateTimeString } from '../../helpers/utility';
 @inject('store')
 @observer
 export default class BottomBar extends Component {
-  static propTypes = {
-    classes: PropTypes.object.isRequired,
```
Comment (dwalintukan) [inline src/components/BottomBar/index.js]: this is required. the `Paper` in the render function uses `classes`.

File: src/components/BottomBar/index.js (hunk 1 above)
Comment (amazingandyyy) [reply src/components/BottomBar/index.js]: good point

File: src/components/BottomBar/index.js (hunk 2 above)
Comment (amazingandyyy) [reply src/components/BottomBar/index.js]: oh ok, I thought we are going to remove all propTypes, aren't we?

Comment (dwalintukan) [reply src/components/BottomBar/index.js]: @amazingandyyy classes is a proptype from the material UI lib. its not something that would go into the store. same goes for things like localization proptypes, etc.

**Pull request merged**